test/unit/scripts/workflow-auto-updater/__snapshots__/lib.test.ts.snap

```diff
@@ -2198,7 +2198,7 @@ jobs:
           ADAPTERS_LIST="$(yarn get-changed-adapters "\${CHANGED_FILES_NAME}")"
 
           # export adapters from this step
-          echo "::set-output name=ADAPTERS_LIST::\${ADAPTERS_LIST}"
+          echo "ADAPTERS_LIST=\${ADAPTERS_LIST}" >> $GITHUB_OUTPUT
```

Comment:
```suggestion
          echo "ADAPTERS_LIST=${ADAPTERS_LIST}" >> $GITHUB_OUTPUT
```

Reply:
we need this, this is in the snapshot file